src/BlazorUI/Bit.BlazorUI.Extras/Styles/extra-components.scss

```
@@ -1,4 +1,5 @@
-﻿@import "../Components/AppShell/BitAppShell.scss";
+﻿@import "../Components/AccordionList/BitAccordionList.scss";
```

Stylelint (17.12.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)